awx/playbooks/project_update.yml

```diff
@@ -115,7 +115,7 @@
         - update_insights
 
     - name: Repository Version
-      debug: msg="Repository Version {{ scm_version }}"
+      debug: msg= "Repository Version {{ scm_version }}"
```

Comment:
We should revert this back, it's cleanup from a recent ansible lint PR that @Spredzy landed.

Reply:
Agree on this one, not sure how it landed.

Reply:
clarification for me, take the space out, leave the space in?

Reply:
Whichever passes CI, specifically the style linter?  I suspect "no space" is the correct answer.

Reply:
`yamllint -s awx/playbooks/project_update.yml`

Reply:
ah, gotcha, I will fix